IntelSiliconPkg/Feature/VTd/IntelVTdDxe/DmaProtection.c

```diff
@@ -575,7 +575,8 @@ AcpiNotificationFunc (
   IN VOID       *Context
   )
 {
-  EFI_STATUS  Status;
+  EFI_STATUS          Status;
+  EFI_HANDLE          Handle;  //MU_CHANGE

```

Comment:
nit: The old spacing was fine, no need to change Status here and Handle can sit and same spacing.